src/js/types/trezor/protobuf.js

```diff
@@ -578,26 +578,19 @@ export type CardanoPublicKey = {
 
 export type CardanoSignedTx = {
     tx_hash: string;
-    tx_body: string;
+    serialized_tx: string;
 };
 export type CardanoTxInput = {
     tx_hash: string;
     address_n: Array<number>;
     output_index: number;
-    type?: number;
 };
 export type CardanoTxOutput = {
     address?: string;
     address_n?: Array<number>;
     amount: string;
```

Comment:
I see the amount is a string (apparently because javascript's integer cannot accommodate a full int64) - shouldn't we have the fee as a string as well for consistency (and in the unlikely case it overflows?)

Reply:
Makes sense. I've done some tests with large numbers and it seemed to run alright (it failed because I've hit the fee limit on FW). I've changed it to string anyways - better safe than sorry. It also seems like a good idea for consistency's sake.